Src/BootCamp.Chapter/BalanceStats.cs

```diff
@@ -1,37 +1,224 @@
-﻿namespace BootCamp.Chapter
+﻿using System;
+using System.Globalization;
+using System.Linq;
+using System.Text;
+
+namespace BootCamp.Chapter
 {
     public static class BalanceStats
     {
         /// <summary>
         /// Return name and balance(current) of person who had the biggest historic balance.
-        /// </summary>
-        public static string FindHighestBalanceEver(string[] peopleAndBalances)
+        /// </summary>       
+        public static string FindHighestBalanceEver(string[] peoplesBalances)
         {
-            return "";
+            if (IsNullOrEmpty(peoplesBalances))
+
+                return "N/A.";
```

Comment:
N/A might change, but the concept of some invalid value being returned will not. In order to avoid changing the exact value in multiple places- please use a constant.